tests/integration/ha_tests/test_ha.py

```diff
@@ -67,14 +77,25 @@ async def change_logging(ops_test: OpsTest):
     await set_log_level(ops_test, -1, "replication.election")
 
 
+@pytest.fixture(scope="module")
```

Comment:
minor suggestion: you can place these fixtures in a `conftest.py` file in this working directory (so as to avoid specifying `scope=module`)

Reply:
Scope controls when the fixture will be setup and tore down, so that if we add more tests depending on chaos mesh we don't need to add and remove it multiple times (within the scope).  I don't think moving it to `conftest.py` would work the same.